crates/turbopack-build/src/chunking_context.rs

```diff
@@ -51,12 +51,12 @@ pub enum MinifyType {
     NoMinify,
 }
 
-/// A builder for [`Vc<BuildChunkingContext>`].
-pub struct BuildChunkingContextBuilder {
-    chunking_context: BuildChunkingContext,
+/// A builder for [`Vc<NodeJsChunkingContext>`].
+pub struct NodeJsChunkingContextBuilder {
```

Comment:
Wondering if there's a better name, given that this includes edge (afaik?)

Reply:
Edge uses the BrowserChunkingContext: https://github.com/vercel/next.js/blob/canary/packages/next-swc/crates/next-core/src/next_edge/context.rs#L160